src/libsync/syncfilestatustracker.cpp

```diff
@@ -112,6 +91,7 @@ static inline bool hasExcludedStatus(const SyncFileItem &item)
 
 SyncFileStatusTracker::SyncFileStatusTracker(SyncEngine *syncEngine)
     : _syncEngine(syncEngine)
+    , _caseSensitivity(Utility::fsCasePreserving() ? Qt::CaseInsensitive : Qt::CaseSensitive)
```

Comment:
This snippet is duplicated above. You might want to implement a little utility function instead that wraps `fsCasePreserving` and returns the appropriate enum entry.